docs/source/how-to/developing-substra/index.rst

```diff
@@ -0,0 +1,9 @@
+How-to guides for Substra developers
+====================================
```

Comment:
The titles for these sections are

- How-to guides for Substra developers
- How-to guides for deploying Substra
- How-to guides for Substra users

Do you think we could standardize on one formulation? such as "for developing on Substra" and "for using Substra". (unfortunately "deployers" isn't a word.. But I think it might be better to focus on action than on title)